sample_install_only_java_env.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# The LabKey Install-Script is broken out into to step functions.  Each STEP can be skipped by exporting the
+# corresponding environment variable with the following syntax:
+# export LABKEY_INSTALL_SKIP_<<Function_NAME>>_STEP=1
+#         or
+# export LABKEY_INSTALL_SKIP_DOWNLOAD_STEP=1
+# to skip the function "step_download()"
+#
+# Sample Env vars to install/update only Java
+
+# Define required defaults - If you installed LabKey with this script previously, these VARS should match your previous
+# installation settings.
+export LABKEY_APP_HOME="/labkey"
+export LABKEY_FILES_ROOT="/labkey/labkey/files"
+export LABKEY_COMPANY_NAME="LabKey"
+export LABKEY_SYSTEM_DESCRIPTION="labkey demo deployment"
+export LABKEY_BASE_SERVER_URL="https://localhost"
+export TOMCAT_INSTALL_HOME="${LABKEY_APP_HOME}/apps/tomcat"
+export TOMCAT_INSTALL_TYPE="Standard"
+
+# Skip installation steps except those needed to install the OS pre-requisites - this function installs java and other
+# os specific dependencies.
+# Need Intro and Default ENVS
+#export LABKEY_INSTALL_SKIP_INTRO_STEP=1
```

Comment:
If they're needed, why list them here commented out? Just remove the lines entirely?